master/internal/command/command.go

```diff
@@ -321,12 +321,22 @@ func (c *command) Receive(ctx *actor.Context) error {
 			ctx.Respond(err)
 		}
 
+	case job.SetResourcePool:
+		err := c.setResourcePool(ctx, msg)
+		if err != nil {
+			ctx.Respond(err)
+		}
+
 	default:
 		return actor.ErrUnexpectedMessage(ctx)
 	}
 	return nil
 }
 
+func (c *command) setResourcePool(ctx *actor.Context, msg job.SetResourcePool) error {
```

Comment:
if we're going with a sep fn like this. how about having the arg be the resourcepool name?

Reply:
sure that makes sense. This function is mostly to set up future implementation.